plugins/tiddlywiki/googleanalytics/readme.tid

```diff
@@ -2,4 +2,5 @@ title: $:/plugins/tiddlywiki/googleanalytics/readme
 
 This plugin enables you to use Google Analytics to track access to your online TiddlyWiki document.
 
-[[Source code|https://github.com/Jermolene/TiddlyWiki5/blob/master/plugins/tiddlywiki/googleanalytics]]
+* [[Current plugin source|https://framagit.org/sycom/TiddlyWiki5/tree/plugin-googleanalytics]]
```

Comment:
Definitely not. First of my mistakes :-/ I did just as my merge is already rejected... This one can be blamed...